examples/tf-azurerm-computegroup/README.md

```diff
@@ -0,0 +1,70 @@
+Deploys a group of Virtual Machines exposed to a public IP via a Load Balancer
+==============================================================================
+
+This Terraform module deploys a Virtual Machines Scale Set in Azure with the following characteristics: 
+
+- Creates a virtual network and a subnet using the 10.0.0.0/16 and 10.0.1.0/24 address space.
+- Creates a load balancer for the scale set of virtual machines
+- Exposes through NAT one or several ports of the VMs on the load balancer
+
+Module Input Variables 
+----------------------
+
+- `resource_group_name` - The name of the resource group in which the resources will be created.
+- `location` - The Azure location where the resources will be created.
+- `vn_size` - The initial size of the virtual machine that will be used in the VM Scale Set.
+- `admin_username` - The name of the administrator to access the machines part of the virtual machine scale set. 
+- `admin_password` - The password of the administrator account. The password must comply with the complexity requirements for Azure virtual machines.
+- `ssh_key` - The path on the local machine of the ssh public key in the case of a Linux deployment.  
```

Comment:
I've been thinking if we should default to the existing one or not, I have the answer :)